test-cypress/integration/import/import-view.spec.js

```diff
@@ -98,6 +98,8 @@ describe('Import view', () => {
 
         // When I go to user tab and upload a file
         ImportSteps.openUserDataTab();
+        // And I wait for tab data to load
+        cy.wait(100);
```

Comment:
Does this improve the things?

Reply:
Locally, the test keeps passing. Seems like it gives Cypress just enough time to load what it needs on the page and find the modal on file upload. I tried checking for file creation and size and contents instead of waiting, but it was ineffective (the file was created, but the test still failed).